NuGet.Config

```diff
@@ -8,11 +8,5 @@
     <clear />
 
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="perfolizer-nightly" value="https://www.myget.org/F/perfolizer/api/v3/index.json" />
-    <!-- reuquired to run Mono Wasm benchmarks -->
```

Comment:
these comments make it clear why we need these feeds or I am missing something?

Reply:
I thought if everything restores correctly then it should be fine?

My assumption when I opened this PR was that they were needed at that time, but now the needed packages were pushed to NuGet.org.

Reply:
At the very least, perfolizer feed can be removed since the version used is already on NuGet.org

https://github.com/dotnet/BenchmarkDotNet/blob/64c3a3cb4ad4057b6065a2783a4b333206f8b5e5/src/BenchmarkDotNet/BenchmarkDotNet.csproj#L23

Reply:
@Youssef1313 you are right, perfolizer-nightly can be safely removed.

Reply:
> I thought if everything restores correctly then it should be fine?

BenchmarkDotNet builds code when running benchmarks, so you don't need Mono AOT feeds until you run Mono AOT benchmarks. You would need to cleanup your local nuget cache and run Mono AOT benchmarks for .NET 6 and 7 to ensure that everything is fine when the feeds are gone..

Reply:
Thanks for clarifying @AndreyAkinshin @adamsitnik 

I sent you an email to discuss this further